src/sentry/api/endpoints/organization_artifactbundle_assemble.py

```diff
@@ -0,0 +1,81 @@
+import jsonschema
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.organization import OrganizationReleasesBaseEndpoint
+from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.tasks.assemble import (
+    AssembleTask,
+    ChunkFileState,
+    get_assemble_status,
+    set_assemble_status,
+)
+from sentry.utils import json
+
+
+@region_silo_endpoint
+class OrganizationArtifactBundleAssembleEndpoint(OrganizationReleasesBaseEndpoint):
+    def post(self, request: Request, organization) -> Response:
+        """
+        Assembles an artifact bundle and stores the debug ids in the database.
+        """
+        if not self.has_release_permission(request, organization, None):
+            raise ResourceDoesNotExist
+
+        # TODO: get real project id from HTTP request.
+        project_id = 0
+
+        schema = {
+            "type": "object",
+            "properties": {
```

Comment:
I would put an `projects` key into the schema and then allow the request to specify one or more projects for association. That would also mean that `assemble_artifacts` takes a list of project IDs rather than one.

Reply:
Yep, I still didn't implement the new schema for the endpoint